site/en/docs/extensions/reference/declarativeNetRequest/index.md

```diff
@@ -148,6 +152,22 @@ is determined based on the priority of each rule and the operations specified.
   `append` rules from the same extension.
 - If a rule has removed a header, then lower priority rules cannot further modify the header.
 
+### Indexing
+
+Static rulesets are indexed by the browser before they can be used. For large numbers of static
+rules, indexing can take some time.
+
+For packed extensions, the browser defers indexing static rulesets until the ruleset is enabled.
+This avoids unnecessarily indexing disabled static rulesets and ensures a minimal extension install
+time for the user. Invalid static rules are ignored.
+
+For unpacked extensions, all static rulesets are indexed when the extension is installed or
+updated. The browser displays extension installation errors or extension warnings for any invalid
+static rules - even for rules in disabled static rulesets.
+
```

Comment:
This goes into internal implementation details. I wonder if we can just say something like:

"Invalid static rules are ignored for packed extensions. For unpacked extensions, the browser displays errors and warnings during installation for any invalid
static rules - even for rules in disabled static rulesets. 

When using static rulesets, it's therefore important to test an unpacked version of your extension. This helps ensure that all of the static rules are valid."

wdyt?

Reply:
Yea, you're right. No need to go into so many irrelevant implementation details. Done.